AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts

```diff
@@ -14,4 +16,6 @@ export class ParticipantModel {
     company?: string | undefined;
     is_judge: boolean;
     is_exist_person: boolean;
+    interpreterFor?: string | undefined;
```

Comment:
`undefined` isn't needed for optional members

Reply:
updated to string